File: src/baekjoon/java/greedy/greedy_2720/Main.java
package baekjoon.java.greedy.greedy_2720;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(br.readLine());
        int quarter = 25;
        int dime = 10;
        int nickel = 5;
        int penny = 1;
        int[][] answer = new int[n][4];
        for(int i=0;i<n;i++){
            int price = Integer.parseInt(br.readLine());
            if(price>=quarter){
                answer[i][0]=price/quarter;
                price = price%quarter;
            }if(price>=dime){
                answer[i][1]=price/dime;
                price = price%dime;
            }if(price>=nickel){
                answer[i][2]=price/nickel;
                price = price%nickel;
            }if(price>=penny){
                answer[i][3]=price/penny;
                price = price%penny;
            }
        }
        for(int i=0;i<n;i++){
            for(int j=0;j<4;j++){
                System.out.print(answer[i][j]+" ");
            }
            System.out.println();
        }
    }
}

File: src/baekjoon/greedy/greedy_2720/Main.kt
package baekjoon.greedy.greedy_2720

import java.util.Scanner


fun main() {
    var sc = Scanner(System.`in`)
    val n = sc.nextInt()
    val array: Array<Array<Int>> = Array(n) { Array(4) { 0 } }
    for (i in 0 until n){

        var num = sc.nextInt()

        var quarter = num / 25
        num %= 25

        var dime = num / 10
        num %= 10

        var nickel = num / 5
        num %= 5

        var penny = num / 1
        num %= 1

        array[i][0] = quarter
        array[i][1] = dime
        array[i][2] = nickel
        array[i][3] = penny
    }
    for(i in 0 until n){
        for (j in 0..3){
            print(array[i][j])
            print(" ")
        }
        println()
    }

}